project/Protobuf.scala

```diff
@@ -62,61 +84,77 @@ object Protobuf {
+    val res = callProtoc(protoc, Seq("--version"), log, { (p, l) =>
+      p !! l
+    })
     val version = res.split(" ").last.trim
     if (version != protocVersion) {
-      sys.error("Wrong protoc version! Expected %s but got %s" format (protocVersion, version))
+      sys.error("Wrong protoc version! Expected %s but got %s".format(protocVersion, version))
     }
   }
 
   private def generate(protoc: String, srcDir: File, targetDir: File, log: Logger, importPath: Option[File]): Unit = {
     val protoFiles = (srcDir ** "*.proto").get
     if (srcDir.exists)
       if (protoFiles.isEmpty)
-        log.info("Skipping empty source directory %s" format srcDir)
+        log.info("Skipping empty source directory %s".format(srcDir))
       else {
         targetDir.mkdirs()
 
         log.info("Generating %d protobuf files from %s to %s".format(protoFiles.size, srcDir, targetDir))
-        protoFiles.foreach { proto => log.info("Compiling %s" format proto) }
+        protoFiles.foreach { proto =>
+          log.info("Compiling %s".format(proto))
+        }
 
         val protoPathArg = importPath match {
-          case None => Nil
+          case None    => Nil
           case Some(p) => Seq("--proto_path", p.absolutePath)
         }
 
-        val exitCode = callProtoc(protoc, Seq("-I" + srcDir.absolutePath, "--java_out=%s" format targetDir.absolutePath) ++
-          protoPathArg ++ protoFiles.map(_.absolutePath), log, { (p, l) => p ! l })
+        val exitCode = callProtoc(
+          protoc,
+          Seq("-I" + srcDir.absolutePath, "--java_out=%s".format(targetDir.absolutePath)) ++
+          protoPathArg ++ protoFiles.map(_.absolutePath),
+          log, { (p, l) =>
+            p ! l
+          })
         if (exitCode != 0)
-          sys.error("protoc returned exit code: %d" format exitCode)
+          sys.error("protoc returned exit code: %d".format(exitCode))
       }
   }
 
   /**
    * Create a transformed version of all files in a directory, given a predicate and a transform function for each file. From sbt-site
    */
-  private def transformDirectory(sourceDir: File, targetDir: File, transformable: File => Boolean, transform: (File, File) => Unit, cache: File, log: Logger): File = {
-    val runTransform = FileFunction.cached(CacheStoreFactory(cache), FilesInfo.hash, FilesInfo.exists) { (in: ChangeReport[File], out: ChangeReport[File]) =>
-      val map = Path.rebase(sourceDir, targetDir)
-      if (in.removed.nonEmpty || in.modified.nonEmpty) {
-        log.info("Preprocessing directory %s..." format sourceDir)
-        for (source <- in.removed; target <- map(source)) {
-          IO delete target
-        }
-        val updated = for (source <- in.modified; target <- map(source)) yield {
-          if (source.isFile) {
-            if (transformable(source)) transform(source, target)
-            else IO.copyFile(source, target)
+  private def transformDirectory(
```

Comment:
intellij with format on save scalafmt is ver insistent on reformatting this, our build only checks source code is formatted not build files